ipld/car/src/error.rs

```diff
@@ -0,0 +1,34 @@
+// Copyright 2020 ChainSafe Systems
+// SPDX-License-Identifier: Apache-2.0, MIT
+
+use cid::multihash::DecodeOwnedError;
+use std::fmt;
+
+#[derive(Debug)]
+pub enum Error {
+    ParsingError(String),
+    InvalidFile(String),
+    Other(String),
+}
+
+impl fmt::Display for Error {
+    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+        match self {
+            Error::ParsingError(err) => write!(f, "Failed to parse CAR file: {}", err.clone()),
+            Error::InvalidFile(err) => write!(f, "Invalid CAR file: {}", err.clone()),
+            Error::Other(err) => write!(f, "Other cid Error: {}", err.clone()),
```

Comment:
nit, this error message was probably copied from the Cid error ?
```suggestion
            Error::Other(err) => write!(f, "CAR Error: {}", err.clone()),
```
or something like that, idk these error variants don't make much sense to me as they are anyway